src/main/java/org/example/learningsystem/core/security/converter/JwtConverter.java

```diff
@@ -0,0 +1,59 @@
+package org.example.learningsystem.core.security.converter;
```

Comment:
I recommend to place this converter in `org.example.learningsystem.btp.xsuaa.converter` as it's XSUAA-specific converter (other auth servers can generate JWT with another structure and we will have to implement dedicated converter for it)